### ISSUE TITLE
Allow resolve upon sdk startup

### ISSUE THREAD

**Issue by akinwale** [lbryio/lbry#2029]
As part of the effort to improve app startup speeds, we need to be able to resolve claims immediately the daemon starts instead of having to wait until the stream manager and wallet components start up.

It would be useful to have flag in the `startupStatus` call which indicates whether or not `resolve` can be successfully called.

**Comment by tzarebczan** [comment]
The wallet connection would need to be established (+ headers downloaded) in order to resolve. We should be able to resolve before loading the wallet/syncing addresses/loading file data.

**Comment by akinwale** [comment]
Agreed. We ultimately have to wait for the headers to be downloaded on first run, so that's fine.

**Comment by lyoshenka** [comment]
@akinwale can we close this?

**Comment by akinwale** [comment]
I don't think it's done. I feel like we should still have this be able to work if we want to be able to have the mobile app use only the SDK, which would be useful in cases where lbry.tv is inaccessible.
